composer.lock

```diff
@@ -5164,36 +5164,37 @@
         },
         {
             "name": "symfony/console",
-            "version": "v4.4.26",
+            "version": "v4.4.27",
```

Comment:
This version is buggy, please update to 4.4.29, or rollback

Reply:
@ihor-sviziev  thank you for the info! I've updated PR, please re-check